pynest/examples/eprop_plasticity/online_eprop_supervised_regression_neuromorphic_mnist.py

```diff
@@ -23,16 +23,17 @@
 Tutorial on learning mnist classification with e-prop
 -------------------------------------------------------
 
-Training a classification model using supervised e-prop plasticity to accumulate evidence.
+Training a classification model using supervised e-prop plasticity to classify the Neuromorphic MNIST data set.
```

Comment:
```suggestion
Training a classification model using supervised e-prop plasticity to classify the Neuromorphic MNIST (N-MNIST) dataset.
```
Use "dataset" instead of "data set" for consistency with the rest of the document.

Reply:
Maybe refer for the remainder of the text to the dataset as the short form "N-MNIST dataset".